core/new-gui/src/app/workspace/component/product-tour/product-tour.component.ts

```diff
@@ -8,7 +8,22 @@ import { TourService, IStepOption } from 'ngx-tour-ng-bootstrap';
 })
 export class ProductTourComponent implements OnInit {
 
-  constructor(public tourService: TourService) { }
+  public loaded = false;
+
+  constructor(public tourService: TourService) {
+    this.tourService.end$.subscribe(() => {
+      location.reload();
```

Comment:
Why do we refresh the page after the tour finishes?

Reply:
After the whole tutorial steps, the workflow that we formed as the example will remain on screen if we do not refresh the page after the tour finished.

Reply:
It's open for discussion but I think we shouldn't refresh the page automatically. The user might want to play with the workflow around for some more time. Let's remove the refresh for now and discuss it again after everything's finished.